### PR TITLE
Commiting the playground app as a simple scaffolding app.

### LINKED ISSUE #2125: [Onboarding] Minimal "Hello World" demo
The [Universal.SampleApp](https://github.com/Microsoft/react-native-windows/tree/rnwcpp-preview/RNWCPP/Universal.SampleApp) is a good starting point for a react-native-windows app, but we need a minimal Hello World app that we can convert into a template for generating new `react-native-windows` projects.

This would be analogous to the project that gets generated by the `react-native windows` command for C#. For reference, the [Playground](https://github.com/Microsoft/react-native-windows/tree/master/ReactWindows/Playground) project is roughly equivalent to the template.

### PR DESCRIPTION
**Pull request by harinikmsft**
Fixes #2125 

Added a simple Playground app with references to minimal projects (ReactUWP, ReactCommon, ReactWindowsCore, Folly) needed to build RN for UWP using RNWCPP. The index.uwp.js file contains the basic RNJS template for developers to edit. 

TODO: 
- The sample app can use one more round of clean up to wrap a lot of the SampleModules, instance properties etc., into some interface. For now, I have used whatever was minimally needed to get it running.
- Add in-app debugging tools once the infrastructure for it has been added in the C++ implementation

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/Microsoft/react-native-windows/pull/2244)

### CODE REVIEW

**Comment by acoates-ms** [comment]
We should probably add this to the list of solution files that gets built as part of the CI loop.
See RNWCPP\.ado\windows-vs-pr.yml

**Comment by jonthysell** [comment]
I kicked off the build, but the yml doesn't even build the playground app. Will try again.

File: RNWCPP/Playground/Playground/MainPage.xaml.cs [hunk 1]
```diff
@@ -72,17 +72,17 @@ public SampleModule()
```
Comment (kmelmon) [inline RNWCPP/Playground/Playground/MainPage.xaml.cs]: nit:  indenting

File: RNWCPP/Playground/Playground/MainPage.xaml.cs [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+      RootElement.InitialProps = initialProps;
+
+      RootElement.JsComponentName = JSCOMPONENTNAME;
+
+      RootElement.StartRender();
+    }
+  }
+
+  public class SampleModule : react.uwp.IModule
```
Comment (kmelmon) [inline RNWCPP/Playground/Playground/MainPage.xaml.cs]: >SampleModule [](start = 15, length = 12)

Will this code ever do something useful?  If not, remove.  We should probably have separate samples for native modules/custom views.

**Comment by kmelmon** [comment]
sorry I should have left the PR open as I made a couple comments.  @jonthysell can you follow up on those?

**Pull request merged**